services/api/src/resources/task/resolvers.ts

```diff
@@ -125,7 +126,30 @@ export const getTasksByEnvironmentId: ResolverFn = async (
     queryBuilder = queryBuilder.limit(limit);
   }
 
-  return query(sqlClientPool, queryBuilder.toString());
+  const rows = await query(sqlClientPool, queryBuilder.toString())
+
+  let adminTasks = false
+  // do a check of all returned tasks for any that are admin tasks
+  for (const row of rows) {
+    if (row.adminOnlyView == true) {
+      adminTasks = true
```

Comment:
I think that to be consistent with line 147, maybe add a semicolon (I'm not sure where we stand on ASI with ts, but I guess consistency in the same block of code is good.